### PR TITLE
Radha

### PR DESCRIPTION
**Pull request by RadhaArul**
completed sorting and filtering together in Memberoverviewindex

### CODE REVIEW

File: Garage 2.0/Controllers/MembersController/MembersController.cs [hunk 1]
```diff
@@ -25,10 +25,21 @@ public async Task<IActionResult> Index()
...
             ViewData["CurrentSort"] = sortOrder;
             ViewBag.NameSortParm = String.IsNullOrEmpty(sortOrder) ? "FirstName_desc" : "";
+            if (ssn != null)
+            {
+                pageNumber = 1;
+            }
+            else
+            {
+                ssn = currentFilter;
+            }
```
Comment (cisci86) [inline Garage 2.0/Controllers/MembersController/MembersController.cs]: Could you add a the viewBag.Button here? 
I only want the button on the end of the memberOverviewIndex page to show when the user has done a search.

File: Garage 2.0/Views/Members/MemberOverviewIndex.cshtml [hunk 2]
```diff
@@ -89,18 +86,20 @@
...
+   asp-route-currentFilter="@ViewBag.CurrentFilte"
    asp-route-pageNumber="@(Model.PageIndex + 1)"
    class="btn btn-default text-primary @nextDisabled">
     Next
 </a>
-@if (ViewBag.Button != null)
+@*@if (ViewBag.Button != null)
 {
     <a asp-action="Index" class="btn btn-primary">Go Back</a>
-}
```
Comment (cisci86) [inline Garage 2.0/Views/Members/MemberOverviewIndex.cshtml:105]: The asp-action should be memberOverviewIndex insted of Index. I forgot to change that yesterday

File: Garage 2.0/Controllers/MembersController/MembersController.cs (hunk 1 above)
Comment (RadhaArul) [reply Garage 2.0/Controllers/MembersController/MembersController.cs]: i removed all extra viewbags because sorting and searching are interconnected

Comment (RadhaArul) [reply Garage 2.0/Controllers/MembersController/MembersController.cs]: i can show demo. then you can understand

**Review by cisci86 — APPROVED** [review]

**Pull request merged**